src/components/RegionSelect/RegionSelect.module.scss

```
@@ -1,80 +1,174 @@
+  padding-top: 0;
+  padding-bottom: 0;
 }
 
-.group-list {
-  padding: 0;
-  margin: 0;
-  list-style: none;
-  font-size: theme.$textSizeSmall;
-}
+// To be implemented with regional grouping headers
+//.group-header {
```

Stylelint (17.4.0)

[error] Expected a space after // (scss/double-slash-comment-whitespace-inside)

(scss/double-slash-comment-whitespace-inside)